CommonTools/PileupAlgos/interface/PuppiContainer.h

```diff
@@ -39,12 +39,20 @@ class PuppiContainer{
     void initialize(const std::vector<RecoObj> &iRecoObjects);
     std::vector<fastjet::PseudoJet> const & pfParticles() const { return fPFParticles; }    
     std::vector<fastjet::PseudoJet> const & pvParticles() const { return fChargedPV; }        
-    std::vector<double> const & puppiWeights() ;
+    std::vector<double> const & puppiWeights();
+    std::vector<double> const & puppiRawAlphas(){ return fRawAlphas; }
+    std::vector<double> const & puppiAlphas(){ return fVals; }
+    // const std::vector<double> puppiAlpha   () {return fAlpha;}
+    std::vector<double> const & puppiAlphasMed() {return fAlphaMed;}
+    std::vector<double> const & puppiAlphasRMS() {return fAlphaRMS;}
+
+    int puppiNAlgos(){ return fNAlgos; }
```

Comment:
I mean, the ones that don't change the state of PuppiContainer, should be const-qualified